account-kit/plugingen/yarn.lock

```diff
@@ -0,0 +1,543 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is this file expected? I thought we only had a root yarn.lock file

Reply:
Ahh this got added when I was working on a bug - good catch